src/open_inwoner/js/components/contacts/index.js

```diff
@@ -0,0 +1,18 @@
+class Contacts {
+  constructor(node) {
+    this.node = node
+    this.node.addEventListener('click', this.toggleOpen.bind(this))
+  }
+
+  toggleOpen(event) {
+    event.preventDefault()
+    this.node.parentElement.parentElement.classList.toggle(
```

Comment:
For the future: selecting up with a .parent.parent type of lookup is generally not a great pattern. It's better to select the outside first and then grab sub-parts, or use a real selector.